libs/langgraph/src/tests/func.test.ts

```diff
@@ -274,110 +274,62 @@ export function runFuncTests(
           expect(timeDiff).toBeGreaterThanOrEqual(timeDelay);
         });
 
-        describe("generator functions as entrypoints", () => {
-          it("can use a generator as an entrypoint", async () => {
-            // equivalent to `test_entrypoint_generator` in python tests
-            const graph = entrypoint(
-              { name: "graph", checkpointer },
-              async function* () {
-                yield "a";
-                yield "b";
-              }
-            );
-
-            const config = { configurable: { thread_id } };
-
-            const result: ("b" | "a")[] = await graph.invoke({}, config);
-            expect(result).toEqual(["a", "b"]);
-
-            expect(await gatherIterator(graph.stream({}, config))).toEqual([
-              "a",
-              "b",
-            ]);
-          });
-
-          it("can use a stream writer", async () => {
-            // equivalent to `test_entrypoint_request_stream_writer` in python tests
-            const graph = entrypoint(
-              { name: "graph", checkpointer },
-              async function* (_: object, config: LangGraphRunnableConfig) {
-                config.writer?.("a");
-                yield "b";
-              }
-            );
-
-            const config = { configurable: { thread_id } };
-            expect(await graph.invoke({}, config)).toEqual(["b"]);
-            expect(await gatherIterator(graph.stream({}, config))).toEqual([
-              "a",
-              "b",
-            ]);
+        it("can use a stream writer", async () => {
+          const graph = entrypoint(
+            { name: "graph", checkpointer },
+            async (input: string, config: LangGraphRunnableConfig) => {
+              config.writer?.(`hello ${input}`);
+              await new Promise((resolve) => {
+                setTimeout(resolve, 100);
+              });
+              config.writer?.(`hello again, ${input}`);
+              return `goodbye, ${input}`;
+            }
+          );
```

Comment:
@nfcampos @eyurtsev - not worth reviewing this entire PR yet, but just confirming that this behavior is what we're expecting for sans entrypoint generators.

I suppose I should also write a test demonstrating subgraph streaming 🤔